.github/workflows/oclif-release.yml

```diff
@@ -48,14 +49,16 @@ jobs:
         id: sha256
         if: ${{matrix.publish == 'homebrew'}}
         run: |
+          make clean
+          make install
           npx oclif-dev publish
```

Comment:
[self]: oclif-dev publish's alternative is not working as expected, so keeping oclif-dev intact until. Issue raised [here](https://github.com/oclif/oclif/issues/956).